packages/devtools/lib/src/inspector/inspector_tree.dart

```diff
@@ -540,6 +544,74 @@ abstract class InspectorTree {
     });
   }
 
+  String get tooltip;
+  set tooltip(String value);
```

Comment:
is the getter/setter necessary here? why not just have a tooltip var?

Reply:
if you could indicate that a var is abstract that would be fine.
the trick is different implementations will want to implement tooltips differently.

Reply:
Ah got it.